### PR TITLE
MODELDEBUG

### PR DESCRIPTION
**Pull request by MasterX1582**
Added MODELDEBUG to config.php
$debug was previously only accessible from Model.php witch is part of the core framework. Its not advised that framework files are modified so this modification will benefit in the long run.

### CODE REVIEW

**Comment by MasterX1582** [comment]
Never mind, i guess i was too enthusiastic. I'll leave the code here until you've seen it but i'll remove it afterwards.
The code doesn't work but i think you'll get the idea once you see it.

File: config/config.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 //The main config file
 define('BASE_URL', '');
 define('ENV', 'dev');
+define('MODELDEBUG', 'false');
```
Comment (Andrewsuares) [inline config/config.php:5]: Great idea. You need to define it like this. Do not enclose the false in quotes. That would turn the value to a string. Tested it and it works.
define('MODELDEBUG', false);

**Comment by davidjconnelly** [comment]
Sounds interesting.

So are you suggesting that the debug feature should be switched on and off
from config.php?

Regards,

DC

**Comment by davidjconnelly** [comment]
The idea of having all of the config options inside a config folder and perhaps even inside a particular config file is appealing.  I get that.  However, since Trongate is a truly modular framework there will surely be times when we have to adjust settings from other parts of the framework such as within modules.  It's something that we should get used to.  My mind is not 100% made up on this but, for the moment, I think I'm happy with the way it works.  I may take a vote on that in the near future.

**Pull request closed without merging**